doc/user_guide/_docs/B01-comparisons-with-alternatives.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Comparison with Alternatives"
```

Comment:
Probably need to add to the https://github.com/intel/rohd-website/blob/main/docs/_data/navigation.yml to let rohd-website pick this up.

Reply:
Working on adding stuff here: https://github.com/intel/rohd-website/pull/23